### PR TITLE
Remove QGIS-trust test

### PR DESCRIPTION
**Pull request by yjacolin**

### CODE REVIEW

File: archives/header.html [hunk 1]
```diff
@@ -27,7 +27,7 @@ <h2>Sponsor and contributions</h2>
     <li>Camptocamp (main contributors)</li>
     <li>Andrea Aime (GeoServer contributions)</li>
     <li>QGIS (html summary with graph)</li>
-    <li>3Liz (QGIS trust contribution)</li>
+    <li>3Liz (QGIS trust contribution - removed)</li>
```
Comment (pmauduit) [inline archives/header.html:30]: why not completely removing the line (if it points to nothing afterwards) ? we should find it back in the git history anyway

**Review by pmauduit — APPROVED** [review]
Just one non-blocker remark, otherwise looks good

File: archives/header.html (hunk 1 above)
Comment (yjacolin) [reply archives/header.html:30]: good question: I think this is good to keep memory of past contribution.

Comment (pmauduit) [reply archives/header.html:30]: you have it in the git history already ;-)

Comment (yjacolin) [reply archives/header.html:30]: well, I meant a marketing point of view, header.html is a public page

Comment (pmauduit) [reply archives/header.html:30]: fair enough, then

**Pull request merged**